hs_core/templates/search/search.html

```diff
@@ -50,190 +36,515 @@ <h4 class="panel-title">
+                var item_lat = (parseFloat(item.northlimit) + parseFloat(item.southlimit)) / 2;
+                var item_lng = (parseFloat(item.eastlimit) + parseFloat(item.westlimit)) / 2;
+
+                var lat_diff = Math.abs(test_lat - item_lat);
+                var lng_diff = Math.abs(test_lng - item_lng);
+
+                if (lat_diff < minFloatingNumber && lng_diff < minFloatingNumber) {
+                    item.title = item.title + "<br />" + test.title;
+                    item.info_link += '<br />' + '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                    existed = true;
+                }
+            });
+            if (!existed){
+                test.info_link = '"<a href="' + test.get_absolute_url + '">' + test.title + '</a>';
+                modified_boxes_data.push(test);
+            }
+        }
+        var reorderDivs = function() {
+            var faceted_fields = ['author', 'subjects', 'resource_type', 'owners_names', 'availability'];
+            var div_ordering = [];
+            faceted_fields.forEach(function(field){
+                var faceting_div = "faceting-"+field;
+                div_ordering.push(faceting_div);
+            });
+            var i;
+            for (i = 1; i < div_ordering.length; i++) {
+                var div0 = "#" + div_ordering[i-1];
+                var div1 = "#" + div_ordering[i];
+                $(div1).insertAfter(div0);
+            }
+        }
+
+        var copySearchResults = function(search_results){
+            search_results.forEach(function(item){
+                json_results.push(item);
+            });
+        }
+
+        var formURL = function(key, value){
+            var currentURL = window.location.href.split('#')[0];
+            var currentHash = window.location.href.split('#')[1];
+            var requestURL = currentURL + "&selected_facets="+key+"_exact:"+value + "#" + currentHash;
+            map = null;
+            window.location = requestURL;
+            var checkboxId =  key + "-" + value;
+            localStorage[checkboxId] = true;
+        }
+
+        var clearAllFaceted = function() {
+            localStorage.clear();
+            var clearURL = "/search/?q=";
+            window.location = clearURL;
+        }
+
+        var clearLocalStorageCheckboxes = function() {
+            $(".faceted-selections").each(function () {
+                $(this).prop("checked", false);
+                delete localStorage[$(this).attr("id")];
+            });
+        }
+
+
+        $(document).ready(function () {
+
+            /*if($("#id_q").val() == ""){
+                localStorage.clear();
+                var searchURL = "?q=" + "";
+                var windowPath = window.location.pathname;
+                var requestURL =  windowPath + searchURL;
+                window.location = requestURL;
+            }*/
+
+            if (window.location.search.length == 0){
                 localStorage.clear();
```

Comment:
This was supposed to be removed. We cannot clear localStorage used by other parts of the site.